tests/utils/test_process_util.py

```diff
@@ -92,3 +92,21 @@ def test_format_stdout_stderr05(self):
         expected = '[stdout]\n\n\n[stderr]\n'
         actual = format_stdout_stderr('', '', 1000)
         self.assertEqual(expected, actual)
+
+    def test_read_output_valid_stdout_stderr(self):
```

Comment:
Unit tests for this method are already checked in develop: https://github.com/Azure/WALinuxAgent/blob/develop/tests/utils/test_process_util.py#L22

Reply:
I'll remove this test and put the tests in a new PR which would also test non-ascii text.